pkg/logs/config/integration_config.go

```diff
@@ -139,3 +139,20 @@ func (c *LogsConfig) Compile() error {
 	}
 	return nil
 }
+
+func (c *LogsConfig) updateTags(tags []string) {
```

Comment:
can we keep the `LogsConfig` immutable please and move this logic somewhere else and perform a merge collecting tags ?

Reply:
I guess the `Origin` is a good candidate ?